README.md

```diff
@@ -12,7 +12,7 @@
 
 APISIX is a cloud-native microservices API gateway, delivering the ultimate performance, security, open source and scalable platform for all your APIs and microservices.
 
-APISIX is based on OpenResty and etcd. Compared with traditional API gateways, APISIX has dynamic routing and plug-in hot loading, which is especially suitable for API management under micro-service system.
+APISIX is based on OpenResty/Tengine and etcd. Compared with traditional API gateways, APISIX has dynamic routing and plug-in hot loading, which is especially suitable for API management under micro-service system.
```

Comment:
It's too early to support Tengine now.

Reply:
we can test but not support official at this time.

Reply:
All test cases have been passed and are fully supported now.